crates/fuel-core/src/graphql_api/worker_service.rs

```diff
@@ -93,9 +98,16 @@ use std::{
 #[cfg(test)]
 mod tests;
 
+#[derive(Debug, Clone)]
```

Comment:
Could also be `Copy`, because `fuel_core_compression::config::Config` is.

However, when it comes to configs I prefer explicit cloning.